pyproject.toml

```diff
@@ -9,6 +9,7 @@ python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.26.0"
+pytest = "^7.0.0"
```

Comment:
Minor, but this can be a dev-dependency. That means when we deploy the app later we can avoid installing pytest since it's not required for the app itself, just the tests.